src/Controllers/FrontEnd/EditCoursePageController.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Portal\Controllers\FrontEnd;
+
+use Portal\Abstracts\Controller;
+use Portal\Models\CourseModel;
+use Portal\Models\TrainerModel;
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Slim\Views\PhpRenderer;
+
+class EditCoursePageController extends Controller
+{
+    private CourseModel $courseModel;
+    private PhpRenderer $renderer;
+    private TrainerModel $trainerModel;
+
+    public function __construct(CourseModel $courseModel, PhpRenderer $renderer, TrainerModel $trainerModel)
+    {
+        $this->courseModel = $courseModel;
+        $this->renderer = $renderer;
+        $this->trainerModel = $trainerModel;
+    }
+
+    public function __invoke(Request $request, Response $response, array $args): Response
+    {
+
+        if (!empty($_SESSION['loggedIn']) && $_SESSION['loggedIn']) {
+            $id = $args['id'];
+            if ($id > 0) {
```

Comment:
`$id` is actually going to be a string at this point, so this comparison doesn't really make sense